adaptivefiltering/dataset.py

```diff
@@ -265,11 +267,15 @@ def __init__(self, dataset=None, **rasterization_options):
             )
 
         # Create the model by running the pipeline
-        execute_pdal_pipeline(
-            dataset=self.dataset,
-            config=config,
-        )
-
+        try:
+            execute_pdal_pipeline(
+                dataset=self.dataset,
+                config=config,
+            )
+        except RuntimeError:
+            raise AdaptiveFilteringError(
+                "The writers.raster was not able to generate a raster. Did you specify a classification that is not present in the dataset?"
```

Comment:
`writers.raster` -> `GDAL`. This is meant for users without knowledge of the inner workings.